suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -397,13 +397,43 @@ export const startDiscoveryThunk = createThunk(
         }
 
         // prepare bundle of accounts to discover, exclude unsupported account types
-        const bundle = await dispatch(
+        const bundleWithSol = await dispatch(
             getBundleThunk({
                 discovery: { ...discovery, availableCardanoDerivations },
                 device,
             }),
         ).unwrap();
 
+        const solanaDiscoveryItem = bundleWithSol.find(
```

Comment:
would be nice to wrap it in code comment mentioning that it will be reverted, but it's just a nitpick, it has to be reverted to make it really work anyway...

Reply:
thanks